checksum.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+RESULT_FILE=$1
+
+if [ -f $RESULT_FILE ]; then
+  rm $RESULT_FILE
+fi
+touch $RESULT_FILE
+
+checksum_file() {
+  echo $(openssl md5 $1 | awk '{print $2}')
+}
+
+FILES=()
+while read -r -d ''; do
+	FILES+=("$REPLY")
+done < <(find . -type f \( -name "build.gradle*" -o -name "dependencies.kt" -o -name "gradle-wrapper.properties" \) -print0)
```

Comment:
Do we have a `dependencies.kt` in this tree?

Reply:
Good find, that’s probably unnecessary here.